src/Shared/INodePacketFactory.cs

```diff
@@ -1,10 +1,5 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
```

Comment:
nit: Is it intentional to have no empty line between the copyright header and the namespace declaration?